utils/steward-cli/steward_cli_notes.md

```diff
@@ -0,0 +1,296 @@
+
+# Accounts
+
+**Program**
+`sssh4zkKhX8jXTNQz1xDHyGpygzgu2UhcRcUvZihBjP`
+
+**Authority**
+`aaaDerwdMyzNkoX1aSoTi3UtFe2W45vh5wCgQNhsjF8`
+
+**Steward Config**
+`35mMfFNiui7hcHy6xHTz11Q6YukbhH9qQgYR5dhWAQQH`
+
+**Stake Pool**
+`3DuPtyTAKrxKfHkSPZ5fqCayMcGru1BarAKKTfGDeo2j`
+
+**Staker**
+`4m64H5TbwAGtZVnxaGAVoTSwjZGV8BCLKRPr8agKQv4Z`
+
+**State**
+`Hmctj1WrZnBx3cmJ8njeid6zKMGT8XHp8C6UkojSF72C`
```

Comment:
Update these and examples with current addresses for JitoSOL mainnet. Can also link to this doc from the main README

Reply:
Good call